src/daemon/ca/certauth.rs

```diff
@@ -1272,6 +1277,32 @@ impl CertAuth {
 
         Ok(res)
     }
+
+    /// Drop a resource class because it no longer works on this parent for the specified
```

Comment:
"works on this parent" - is "on" the correct term, say rather than "with" or "under" or ??

Reply:
fixed